cmd/root.go

```diff
@@ -95,11 +95,12 @@ func New(config Config) (*cobra.Command, error) {
 	}
 
 	c := &cobra.Command{
-		Use:          name,
-		Short:        description,
-		Long:         description,
-		RunE:         r.Run,
-		SilenceUsage: true,
+		Use:           name,
+		Short:         description,
+		Long:          description,
+		RunE:          r.Run,
+		SilenceUsage:  true,
+		SilenceErrors: true,
```

Comment:
This makes cobra shut up and not print errors by itself. Only we control how we print them.